src/app/pages/wallets/wallet-details/empty-transaction-list/pages-wallet-details-empty-transaction-list.component.scss

```diff
@@ -0,0 +1,19 @@
+@use 'sizes';
+
+.no-transactions-container {
```

Comment:
As it will be a reusable component and not only for transactions those classes should be renamed.